lms/djangoapps/mobile_api/models.py

```diff
@@ -1,3 +1,23 @@
 """
 A models.py is required to make this an app (until we move to Django 1.7)
 """
+# pylint: disable=no-member
+
+from django.db.models.fields import TextField
+
+from config_models.models import ConfigurationModel
+
+
+class MobileApiConfig(ConfigurationModel):
+    """Configuration for the video upload feature."""
+    video_profiles = TextField(
+        blank=True,
+        help_text="A comma-separated list of names of profiles when requesting video data from VAL."
+    )
+
+    @classmethod
+    def get_video_profiles(cls):
+        """
+        Get the list of profiles in priority order when requesting from VAL
+        """
+        return [profile for profile in cls.current().video_profiles.split(",") if profile]
```

Comment:
Consider calling `profile.strip` so any leading/trailing white characters are removed.  Especially since the addition of profiles in this CSV is manually done.